test/fuzz/readMessageText.js

```diff
@@ -0,0 +1,27 @@
+import { FuzzedDataProvider } from '@jazzer.js/core';
```

Comment:
Minor, but for consistency, this file should be name `readMessage**Armored**.js` 🙂